common/meta/api/src/meta_api_test_suite.rs

```diff
@@ -498,6 +499,42 @@ impl MetaApiTestSuite {
         Ok(())
     }
 
+    pub async fn list_database_leader_follower<MT: MetaApi>(
+        &self,
+        leader: &MT,
+        follower: &MT,
+    ) -> anyhow::Result<()> {
+        tracing::info!("--- create db1 and db3 on leader");
+        {
+            let dbs = vec!["db1", "db3"];
+            for db_name in dbs {
+                let req = CreateDatabaseReq {
+                    if_not_exists: false,
+                    db: db_name.to_string(),
+                    engine: "github".to_string(),
+                    options: Default::default(),
+                };
+                let res = leader.create_database(req).await;
+                tracing::info!("create database res: {:?}", res);
+                assert!(res.is_ok());
+            }
+        }
+
+        tracing::info!("--- list databases from follower");
+        {
+            let res = follower.list_databases(ListDatabaseReq {}).await;
+            tracing::debug!("get database list: {:?}", res);
+            let res = res?;
+            assert_eq!(2, res.len(), "database list len is 2");
+            assert_eq!(1, res[0].database_id, "db1 id is 1");
+            assert_eq!("db1".to_string(), res[0].db, "db1.name is db1");
```

Comment:
Next time you just do not need a `to_string()` to compare a `&str` with a `String`.